.github/workflows/run_on_dev.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       - name: install dependencies
         run: |
           cd /workspace/multi-lora-fine-tune
+          pip install -e .
```

Comment:
this is necessary?

Reply:
`pipeline_example_check.py` and `transport_check.py` will use mlora module, so we should make a link first.